### ISSUE TITLE
Routine update of dependencies

### ISSUE THREAD

**Issue by phillxnet** [rockstor/rockstor-core#2868]
Update Poetry managed dependencies within existing pyproject.toml constraints.

```
cd /opt/rockstor
poetry update
```

Issue resolution via resulting updated poetry.lock file.

---

Upstream reference: https://python-poetry.org/docs/basic-usage/#updating-dependencies-to-their-latest-versions

**Comment by phillxnet** [comment]
# Resulting Poetry report

Leap 15.6 X86_64 host
```
Package operations: 0 installs, 3 updates, 0 removals

  • Updating certifi (2024.6.2 -> 2024.7.4)
  • Updating setuptools (70.2.0 -> 70.3.0)
  • Updating django (4.2.13 -> 4.2.14)

Writing lock file
```

**Comment by phillxnet** [comment]
# Notable Changelog references:

- https://github.com/certifi/python-certifi/security/advisories/GHSA-248v-346w-9cwc
- https://setuptools.pypa.io/en/stable/history.html
- https://docs.djangoproject.com/en/5.0/releases/4.2.14/

**Comment by phillxnet** [comment]
Closing as:
Fixed by #2869
